src/viam/sdk/robot/client.cpp

```diff
@@ -69,7 +69,8 @@ void RobotClient::close() {
         t->~thread();
     }
     stop_all();
-    viam_channel.close();
+    viam_channel->close();
+    viam_channel = nullptr;
```

Comment:
[q/nit] I don't know if we should set this to `nullptr`. `ViamChannel` has a bool called `closed` that I would expect to be able to check after calling `close()` on this client.

Reply:
Good catch, thanks! This was an artifact of some testing/debugging around an asan issue, but you're right, it's not necessary. removed.